dizoo/atari/config/serial/spaceinvaders/spaceinvaders_acer_config.py

```diff
@@ -53,39 +53,23 @@
             collector=dict(collect_print_freq=1000, ),
         ),
         eval=dict(evaluator=dict(eval_freq=1000, )),
-        other=dict(replay_buffer=dict(
-            replay_buffer_size=3000,
-        ), ),
+        other=dict(replay_buffer=dict(replay_buffer_size=3000, ), ),
     ),
 )
-main_config = EasyDict(spaceinvaders_acer_config)
+spaceinvaders_acer_config = EasyDict(spaceinvaders_acer_config)
+main_config = spaceinvaders_acer_config
 
 spaceinvaders_acer_create_config = dict(
     env=dict(
         type='atari',
         import_names=['dizoo.atari.envs.atari_env'],
     ),
-    env_manager=dict(type='base'),
-    # env_manager=dict(type='subprocess'),
+    env_manager=dict(type='subprocess'),
     policy=dict(type='acer'),
 )
-create_config = EasyDict(spaceinvaders_acer_create_config)
+spaceinvaders_acer_create_config = EasyDict(spaceinvaders_acer_create_config)
+create_config = spaceinvaders_acer_create_config
 
-# if __name__ == '__main__':
-#     serial_pipeline((main_config, create_config), seed=0)
-
-def train(args):
-    main_config.exp_name='spaceinvaders_acer'+'_ns64_ul64_bs64_rbs3e3_10m_seed'+f'{args.seed}'
-    import copy
-    # in theory: 2441.4 iterations = 10M env steps / (64*64) 
-    # in practice: 2500 iterations = 5M env steps 
-    serial_pipeline([copy.deepcopy(main_config), copy.deepcopy(create_config)], seed=args.seed, max_iterations=5500)
-
-if __name__ == "__main__":
-    import argparse
-    for seed in [0,1,2,3,4]:     
-        parser = argparse.ArgumentParser()
-        parser.add_argument('--seed', '-s', type=int, default=seed)
-        args = parser.parse_args()
-        
-        train(args)
+if __name__ == '__main__':
```

Comment:
add line `# or you can enter ding -m serial -c spaceinvaders_acer_config.py -s 0`